query-node/mappings/content-directory/entity/index.ts

```diff
@@ -259,8 +240,10 @@ async function contentDirectory_EntityRemoved(db: DB, event: SubstrateEvent): Pr
       break
 
     default:
-      throw new Error(`Unknown class name: ${cls.name}`)
+      throw new Error(`Unknown class name: ${knownClass.name}`)
   }
+  const ce = await db.get(ClassEntity, where)
+  if (ce) await db.remove<ClassEntity>(ce)
```

Comment:
Further to our call, given that code at this point is dealing with a known class, it must be in the `ClassEntity` table and `ce` should not be undefined. So this line should be turned into an assertion that `ce` is truthy. Or get/remove operation can be changed to a single db operation to remove and fail if the entity is not found?